src/app/games/lodestone/LodestoneDummyMinimax.ts

```diff
@@ -15,7 +15,8 @@ export class LodestoneDummyMinimax extends Minimax<LodestoneMove, LodestoneState
         return this.flatMapEmptyCoords(state, (coord: Coord): LodestoneMove[] => {
             const moves: LodestoneMove[] = [];
             for (const direction of this.nextDirection(state)) {
-                for (const diagonal of [true, false]) {
+                const orientations: LodestoneOrientation[] = ['diagonal', 'orthogonal'];
+                for (const diagonal of orientations) {
```

Comment:
j'vais demander un truc que je demande pas souvent mais: monolignise :D ?
Et orientation of orientations please :D

Reply:
COMMENT ÇA ! :D Mais oui monoligné et orienté

Reply:
Ah bah non j'peux pas monoligner sinon il me croit pas que 'diagonal' et 'orthogonal' sont des LodestoneOrientation !